.gitignore

```diff
@@ -12,9 +12,14 @@ __pycache__
 .pytest_cache
 .python-version
 *.egg-info
+.venv/
 dist
+.venv
 
 ## Terraform ##
 .terraform
 .terraform.lock.hcl
 .tfvars
+
+## MacOS ##
+.DS_Store
```

Comment:
Please move the `## MacOS ##` section to be in the correct place alphabetically.